Core/scripts/dirac-install.py

```diff
@@ -2559,7 +2560,8 @@ def createSymbolicLink():
   else:
     logNOTICE("Skipping installing DIRAC")
 
-  if cliParams.diracOS:
+  # we install with DIRACOS from v7rX DIRAC release
+  if cliParams.diracOS or int(cliParams.release[1]) > 6:
```

Comment:
I am afraid, this will not work if the project is not DIRAC, e.g. LHCb or CTA. You should rather check for the version of the DIRAC module to be installed

Reply:
Correct! Now it should be fine. It's a bit hacky but works.